names/gaurav7019.md

```diff
@@ -0,0 +1,2 @@
+* https://twitter.com/gaurav05_gaurav
+* https://www.linkedin.com/in/gaurav-agarwal05/
```

Comment:
Hello! thanks for the PR.

There is one formatting change that's needed before we merge this, it should be formatted as mentioned in `point 4` :  https://github.com/buildandtell/learnPR , please add the change and push it to the same branch.

Reply:
I have made the required changes.